hv-rhel7.x/hv/netvsc.c

```diff
@@ -898,7 +898,7 @@ int netvsc_send(struct net_device *ndev,
 	#if (RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(7,1))
 	xmit_more = skb->xmit_more &&
 	#else
-	xmit_more = skb &&
+	xmit_more = 
```

Comment:
Can you please merge all the patches into a single one? We don't need to record every detail in the repo history. :-)